projects/plugins/boost/app/assets/src/js/utils/navigate.ts

```diff
@@ -0,0 +1,3 @@
+export const navigate = ( to: string ) => {
+	window.location.hash = to;
+};
```

Comment:
Don't abstract 1 line of code if it's not reused at the moment, especially if you need to create yet another utils file to achieve this.

Reply:
It is there as a stop gap. I want to be able to replace the logic inside it with react-navigator.